eng/pipelines/docindex.yml

```diff
@@ -13,6 +13,18 @@ jobs:
     pool:
       vmImage: windows-2019
     steps:
+      - task: UsePythonVersion@0
+        displayName: 'Use Python 3.6'
+        inputs:
+          versionSpec: '3.6'
+
+      - template: eng/pipelines/templates/scripts/replace-relative-links.yml@azure-sdk-tools
+        parameters:
+          TargetFolder: .
```

Comment:
Does this support passing in a single file?

Reply:
Unfortunately, no, the script takes directory arguments.